paths/migrate.go

```diff
@@ -0,0 +1,59 @@
+// Copyright (c) 2021 Tailscale Inc & AUTHORS All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// Package paths returns platform and user-specific default paths to
```

Comment:
Only one file in a package (conventionally the one named after the package itself) should have a package comment. Otherwise they get concatenated together in weird ways.

So delete these 3 lines and update the paths.go version if desired.

Reply:
Done